packages/eventual-send/src/message-breakpoints.js

```diff
@@ -0,0 +1,148 @@
+import { getEnvironmentOption } from '@endo/env-options';
+
+const { quote: q, Fail } = assert;
+
+const { hasOwn, freeze, entries } = Object;
+
+/**
+ * This is the external JSON representation, in which
+ * - the outer property name is the class-like tag or '*',
+ * - the inner property name is the method name or '*',
+ * - the value is a non-negative integer countdown or '*'.
+ *
+ * @typedef {Record<string, Record<string, number | '*'>>} MessageBreakpoints
```

Comment:
Maybe document in terms of types, like.

```js
/** @typedef {string | '*'} MatchStringTag docs... */
/** @typedef {string | '*'} MatchMethodName docs... */
/** @typedef {number | '*'} MatchCountdown docs... */ 
```

```suggestion
 * @typedef {Record<MatchStringTag, Record<MatchMethodName, MatchCountdown>>} MessageBreakpoints
```

Reply:
Also, could you please describe explicitly what `MatchCountdown` does?  It is not obvious from reading the other jsdocs whether this is the number of noops until entering the debugger, or the number of debugger traps until it becomes a noop.

Reply:
Done.